tests/rptest/services/admin.py

```diff
@@ -223,6 +223,16 @@ def mark_transaction_expired(self,
         path = f"partitions/{namespace}/{topic}/{partition}/mark_transaction_expired?id={pid['id']}&epoch={pid['epoch']}"
         return self._request("post", path, node=node)
 
+    def delete_partition_from_transaction(self,
```

Comment:
I think we should pass all params here separately because they all are necessary.